src/components/MoviesContainer/MovieList/MovieList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import MovieCard from "./MovieCard";
+import movies from "../../../mock-data";
+
+const MovieList = () => (
+  <ul className="movie-list">
+    {movies.data.map((movie) => (
```

Comment:
```suggestion
    {movies.data.map(({id, title, release_date, poster_path, genres}) => (
```

Just to make code cleaner and get rid of repetitive movie.something